R/convertToEGRET.R

```diff
@@ -0,0 +1,75 @@
+#' Convert loadflex to EGRET object
+#'  
+#' Convert a loadflex object into an EGRET object for plotting.
+#' 
+#' @param intdat data.frame of interpolation data
+#' @param estdat data.frame of estimation data
+#' @param preds data.frame of load predictions
+#' @param meta loadflex metadata object; it must include constituent,
+#' flow, conc.units, custom (station abbreviation: staAbbr)
+#' 
+#' @importFrom dplyr left_join
+#' @importFrom dplyr rename_
+#' @importFrom dplyr mutate
+#' @importFrom lubridate decimal_date
+#' @importFrom EGRET as.egret
+#' 
+#' @export
```

Comment:
This was just leftover from me trying to figure out why the package couldn't find this function - but turns out my documentation wasn't complete. I'm not sure on the second point. I think as it stands, no. But maybe with some tweaking it could. Especially if I spend some time on one your previous points with making some of the input arguments optional. For now, I'll not export this.